index.js

```diff
@@ -54,3 +54,12 @@ nextButton.addEventListener("click", () => {
 });
 
 fetchCharacters();
+
+searchBar.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const formData = new FormData(event.target);
+  const data = Object.fromEntries(formData);
+  console.log(data);
+  searchQuery = data.query;
+  fetchCharacters();
```

Comment:
Well done 🙌🏼 

As you only have one input element here, you could also use `searchQuery = event.target.elements.query.value` to access the value of the input element.

Please remove the `console.log` before proceeding 🙂 
And you might run into a bug here: When you are on another page than the first one and start a new search, the page actual page number is going to stay the same instead of changing back to 1. So it is best to set the page to 1 inside this function.

```suggestion
  const formData = new FormData(event.target);
  const data = Object.fromEntries(formData);

  searchQuery = data.query;
  page = 1;
  fetchCharacters();
```